transmart-rest-api/src/main/groovy/org/transmartproject/rest/marshallers/MDStudySerializationHelper.groovy

```diff
@@ -24,6 +24,7 @@ class MDStudySerializationHelper extends AbstractHalOrJsonSerializationHelper<St
                 id: study.id,
                 studyId: study.name,
                 bioExperimentId: study.bioExperimentId,
+                secureObjectToken: study.secureObjectToken,
```

Comment:
Why is this needed to be exposed?

Reply:
Mainly for data loading scripts to determine which studies are non-public, for loading roles into Keycloak (and which token to use).